src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/hosts/hosts.component.html

```diff
@@ -53,8 +53,8 @@
 
 <ng-template #servicesTpl
              let-value="value">
-  <span *ngFor="let instance of value; last as isLast">
-    <span class="badge badge-background-primary ms-1" >{{ instance }}</span>
+  <span *ngFor="let instance of value">
+    <span class="badge badge-background-primary ms-1" >{{ instance['type'] }}: {{ instance['count'] }}</span>
```

Comment:
Let's try to make variable names meaningful. Sometimes it's really to tough to understand what a chunk of code is doing. Why not being kind to future reviewers (probably ourselves) by self-explaining what code is about?:
```suggestion
  <span *ngFor="let service of services">
    <span class="badge badge-background-primary ms-1" >{{ service['type'] }}: {{ service['count'] }}</span>
```

Reply:
Done.

Reply:
thanks!